content/800-data-platform/050-data-proxy/100-use-data-proxy.mdx

```diff
@@ -212,8 +212,22 @@ The generated Prisma Client has a reduced bundle size because the entire query e
 
 You can adjust the maximum number of database connections of each Data Proxy connection pool.
 
-To do so, in the Prisma Data Platform add the `connection_limit` parameter to the database connection string of the environment in your project.
+To do so, in the Prisma Data Platform, add the `connection_limit` parameter to the database connection string of the environment in your project.
+
+The _default_ value for the `connection_limit` in each Query Engine (QE) instance is _5_. This default value is determined using the calculation (`num_physical_cpus * 2 + 1`), where each QE instance consist of two CPUs.
 
 ```
-postgresql://user:password@host:5432/mydb?connection_limit=60
+prisma://region.prisma-data.com/?api_key=&connection_limit=10
 ```
+
+In the example connection string, the connection limit of _10_ is used as an arbitrary value. However, it's important to understand that this connection limit applies to each individual Query Engine (QE) instance.
+
+To ensure that your database's connections are not exhausted, it is recommended to set an appropriate value for the connection limit. This will help manage the surge in traffic effectively and prevent any connection-related issues.
+
+For example, if the `connection_limit` is set to _10_ and _5_ QE instances are
+created, a total of _50_ connections would be established with the database
+(`connection_limit * QE instances`). It is important to ensure that the database
+has sufficient capacity to handle this load of _50_ connections.
+
+For more detailed information regarding the connection pool size calculation, limits of the Data Proxy and its concurrency
+limits, we recommend referring to the [Concurrency Limits documentation](https://www.prisma.io/docs/data-platform/about/limits-and-status#concurrency-limits-of-the-data-proxy) and the [Recommended connection pool size documentation](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#optimizing-for-parallel-requests:~:text=instance%20of%20PrismaClient-,Recommended%20connection%20pool%20size,-The%20recommended%20connection).
```

Comment:
Where is the last link supposed to point to?

Reply:
Hi Jan, this particular section. In the connection management section > Recommended Pool Size Part. 
[Here](https://www.prisma.io/docs/guides/performance-and-optimization/connection-management#optimizing-for-parallel-requests:~:text=instance%20of%20PrismaClient-,Recommended%20connection%20pool%20size,-The%20recommended%20connection)
![image](https://github.com/prisma/docs/assets/64993082/b4369505-1701-46e7-93b2-589e3537e372)

Reply:
Then I think you can just use the anchor link for that headline instead of linking to another headline but then also highlighting some text somehow.

(There are two very different recommendations for different types of applications though)

Reply:
@janpio replaced with anchor tags for now. In the latest [commit](https://github.com/prisma/docs/pull/4776/commits/fd38ac03dc3aabbfa643c19c4e65e98dfcdb2431)